sdk/python/kfp/compiler/read_write_test.py

```diff
@@ -175,12 +175,12 @@ def test(
         """Tests serialization and deserialization consistency and correctness.
 
         Args:
-            name (str): '{test_group_name}-{test_case_name}'. Useful for print statements/debugging.
-            test_case (str): Test case name (without file extension).
-            test_data_dir (str): The directory containing the test case files.
-            function (str, optional): The function name to compile.
-            read (bool): Whether the pipeline/component supports deserialization from YAML (IR, except for V1 component YAML back compatability tests).
-            write (bool): Whether the pipeline/component supports compilation from a Python file.
+            name : '{test_group_name}-{test_case_name}'. Useful for print statements/debugging.
```

Comment:
Can we trim the leading space before the colon ` :` -> `:`? ([ref](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods))